packages/e2e-tests/src/config/bootstrap.js

```diff
@@ -118,7 +118,7 @@ if ('true' === process.env.CI) {
 }
 
 // Set default timeout for individual expect-puppeteer assertions. (Default: 500)
-setDefaultOptions({ timeout: EXPECT_PUPPETEER_TIMEOUT || 500 });
+setDefaultOptions({ timeout: EXPECT_PUPPETEER_TIMEOUT || 1000 });
```

Comment:
This is a big change. I like it, but can explain the reasoning behind it.

Reply:
In my testing this drastically reduced flakiness when running the tests, even locally.

A bump to a 1s timeout seems justifiable.